From: Daniel Schepler <schepler@debian.org>
Date: Sat, 9 Mar 2013 16:33:14 -0800
Subject: Fix tests/check_sizes.c on x32

Bug-Debian: https://bugs.debian.org/702681
---
 tests/check_sizes.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/tests/check_sizes.c b/tests/check_sizes.c
index 0b9809f..8aa36e5 100644
--- a/tests/check_sizes.c
+++ b/tests/check_sizes.c
@@ -30,7 +30,11 @@ static void ksz_print(const char *type,size_t sz)
   } while(0)
 
 #ifdef __x86_64__
+#ifdef __ILP32__
+#define KSZ(type,sz32,sz64) KSZA(type,sz32)
+#else
 #define KSZ(type,sz32,sz64) KSZA(type,sz64)
+#endif
 #elif defined __i386__
 #define KSZ(type,sz32,sz64) KSZA(type,sz32)
 #else
